gufe/protocols/protocolunit.py

```diff
@@ -22,6 +22,8 @@
     GufeTokenizable, GufeKey, TOKENIZABLE_REGISTRY
 )
 
+from ..storage import ExternalStorage
+
 
 @dataclass
 class Context:
```

Comment:
since we're tinkering here, what's the point of the `Context` object here?  it converts two arguments to one, and we use it once?  can we remove it and just have two `ExternalStorage` arguments (shared/scratch) everywhere?

Reply:
I'm fine either way here.

@dotsdl : is there a reason we need the `Context` object?